dts/bindings/display/solomon,ssd1363-common.yaml

```diff
@@ -90,11 +90,11 @@ properties:
     type: boolean
     description: Turn on display color inverting
 
-  greyscale-enhancement:
+  grayscale-enhancement:
     type: boolean
-    description: Enable greyscale enhancement, undocumented.
+    description: Enable grayscale enhancement, undocumented.
 
-  greyscale-table:
+  grayscale-table:
```

Comment:
change needs to be mentioned in migration guide as its requiring downstream users to make changes to their code